net-im/skype-dbus-mock/skype-dbus-mock-0_pre20181011.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6,3_7} )
+
+inherit python-r1
+
+GIT_COMMIT="3a9e2882ac5c0ad6be3c5cb5c7da008b4cfa51da"
+DESCRIPTION="Mocked systemd dbus interface for skype 8.30+"
+HOMEPAGE="https://github.com/maelnor/skype-dbus-mock"
+SRC_URI="https://github.com/maelnor/${PN}/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND=">=net-im/skypeforlinux-8.30
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	!sys-apps/systemd
```

Comment:
Make this and elogind a hard blocker. Portage can "solve" these soft blocks on its own which might surprise users, hard blocker will stop them if they have systemd or elogind installed.

Rest looks good. You'll need to fix the CI issues too.

Reply:
I think soft blocker is fine here. We really only use hard blockers when something needs to be unmerged prior to building other package. They're PITA to users.

Reply:
(and if someone uses systemd or logind, the appropriate USE flags will bring them in anyway)

Reply:
Ok, so there's nothing to be done here then?  Which USE flags were you referring to?

Reply:
Many packages have USE=systemd and/or USE=elogind, generally to cover optional support. I have no strong opinion whether skype should have those flags as well -- I suppose `|| ()` dep works fine here.